dynawo/sources/Modeler/DataInterface/IIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -1359,5 +1362,29 @@ DataInterfaceIIDM::getStaticParameterBoolValue(const std::string& staticID, cons
   return findComponent(staticID)->getStaticParameterValue<bool>(refOrigName);
 }
 
+void
+DataInterfaceIIDM::copy(const DataInterfaceIIDM& other) {
+  networkIIDM_  = other.networkIIDM_;  // No clone here because iidm network is not copyable
```

Comment:
Why bothering to implement this when it will neither be used nor tested?

Reply:
This function is used during clone API

Reply:
ok, then please add a unit test (however before I would like to make sure that we cannot avoid to clone on dyn-algo side when the number of threads is 1)